### PR TITLE
added effects to data packet, web app

### PR DESCRIPTION
**Pull request by jdicarlantonio**
Now able to specify effects in the web app and send them to the user's SQS queue

### CODE REVIEW

File: companion-app/src/Components/UploadFile.js [hunk 1]
```diff
@@ -16,9 +14,22 @@ let packet = {
...
+      wet: 0.0
+    },
+    equalizer: {
+      apply: false,
+      lowgain: 0.0,
+      midgain: 0.0,
+      highgain: 0.0
+    }
   }
 };
```
Comment (Arent128) [inline companion-app/src/Components/UploadFile.js:34]: This seems to match the format we are going for when it comes to audio playback. We just need to ensure that this JSON format can be parsed with the PacketParser class that Brandon wrote.

**Review by Arent128 — APPROVED** [review]
I can see how the added functionality of the new JSON format requires the need to add more ways to track the added fields. As well as the need to ensure that the values gotten from those fields are formatted into the packet that will be created into S3. We just need to be sure that we can parse the Packet Properly in order to add these new options.

File: companion-app/src/Components/UploadFile.js [hunk 2]
```diff
@@ -35,14 +46,32 @@ class UploadFile extends Component {
...
+          eqapply: "false",
+          eqlowgain: 0.0,
+          eqmidgain: 0.0,
+          eqhighgain: 0.0,
+          echoapply: "false",
+          echodelay: 10.0,
+          echofeedback: 0.0,
+          echodry: 0.0,
+          echowet: 0.0
         };
```
Comment (blim012) [inline companion-app/src/Components/UploadFile.js]: Make sure that each value is sent in as a only strings

**Review by blim012 — COMMENTED** [review]
The packet must only have strings as values so that it is parsed correctly by the python script

File: companion-app/src/Components/UploadFile.js (hunk 2 above)
Comment (Arent128) [reply companion-app/src/Components/UploadFile.js]: Also if we are going to use floats, I will need to change the packet validation

**Review by KeithsCreations — APPROVED** [review]

**Review by blim012 — APPROVED** [review]

**Pull request merged**